examples/evaluations/litellm_vectorizer/compose.yaml

```diff
@@ -0,0 +1,24 @@
+name: pgai 
+services:
+  db:
+    image: timescale/timescaledb-ha:pg17
+    environment:
+      POSTGRES_PASSWORD: postgres
+      OPENAI_API_KEY: ""
```

Comment:
You can do ${OPENAI_API_KEY}

and it will interpolate this from environment variables or a .env file. prevents having to set the key multiple times and accidentally commiting them